test/ts/integration/examples/simple_interest_terms_contract/runners/simple_interest_terms_contract_runner.ts

```diff
@@ -0,0 +1,156 @@
+        const debtOrder = this.debtOrder;
+
+        return kernel.fillDebtOrder.sendTransactionAsync(
+            debtOrder.getCreditor(),
+            debtOrder.getOrderAddresses(),
+            debtOrder.getOrderValues(),
+            debtOrder.getOrderBytes32(),
+            debtOrder.getSignaturesV(),
+            debtOrder.getSignaturesR(),
+            debtOrder.getSignaturesS(),
+            { from: UNDERWRITER },
+        );
+    }
+
+    protected async setupDebtOrder(scenario: RegisterRepaymentScenario | RegisterTermStartScenario) {
+        const {
+            simpleInterestTermsContract,
+            kernel,
+            repaymentRouter,
+            debtTokenContract,
+            dummyREPToken,
+        } = this.contracts;
+
+        const { DEBTOR_1, CREDITOR_1, UNDERWRITER, RELAYER } = this.accounts;
+
+        const termsContractParameters = SimpleInterestParameters.pack(
+            scenario.principalTokenIndex,
+            scenario.principalAmount,
+            scenario.interestRate,
+            scenario.amortizationUnitType,
+            scenario.termLengthUnits,
+        );
+
+        const defaultOrderParams = {
+            creditor: CREDITOR_1,
+            creditorFee: Units.ether(0.002),
+            debtKernelContract: kernel.address,
+            debtOrderVersion: kernel.address,
+            debtTokenContract: debtTokenContract.address,
+            debtor: DEBTOR_1,
+            debtorFee: Units.ether(0.001),
+            expirationTimestampInSec: new BigNumber(
+                moment()
+                    .add(1, "days")
+                    .unix(),
+            ),
+            issuanceVersion: repaymentRouter.address,
+            orderSignatories: { debtor: DEBTOR_1, creditor: CREDITOR_1, underwriter: UNDERWRITER },
+            principalAmount: scenario.principalAmount,
+            principalTokenAddress: dummyREPToken.address,
+            relayer: RELAYER,
+            relayerFee: Units.ether(0.0015),
+            termsContract: simpleInterestTermsContract.address,
+            termsContractParameters,
+            underwriter: UNDERWRITER,
+            underwriterFee: Units.ether(0.0015),
+            underwriterRiskRating: Units.percent(1.35),
+        };
+
+        const orderFactory = new DebtOrderFactory(defaultOrderParams);
+
+        const debtOrder = await orderFactory.generateDebtOrder();
+        const agreementId = debtOrder.getIssuanceCommitment().getHash();
+
+        this.debtOrder = debtOrder;
+        this.agreementId = agreementId;
+
+        await this.setBalances();
+    }
+
+    protected async setBalances() {
+        const {
+            tokenTransferProxy,
+        } = this.contracts;
+
+        const { CONTRACT_OWNER } = this.accounts;
+        const debtOrder = this.debtOrder;
+
+        const token = await DummyTokenContract.at(
+            debtOrder.getPrincipalTokenAddress(),
+            web3,
+            { from: CONTRACT_OWNER, gas: 4712388 },
```

Comment:
Nit: This feels like a bit like "magic" values.   Can we make a `TX_DEFAULTS` constant and use that instead?